library/general/allowedrepos/samples/repo-must-be-openpolicyagent/example_disallowed_initcontainer.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
```

Comment:
do we need this still? does `both` scenario cover this too?

Reply:
We do need it. This test case has a positive case and a negative case where the initcontainer is not pulling allowed repo, whereas `both` has 2 violations for both init and container and `all` has 3 violations for init, container, and ephemeral.